modules/aioseop_sitemap.php

```diff
@@ -2033,6 +2049,9 @@ public function do_build_sitemap( $sitemap_type, $page = 0, $filename = '', $com
 			if ( ( 'root' === $sitemap_type ) && ! empty( $this->options[ "{$this->prefix}indexes" ] ) ) {
 				return $this->build_sitemap_index( $sitemap_data, sprintf( $comment, $filename ) );
 			} else {
+				if ( empty( $sitemap_data ) ) {
+					return;
```

Comment:
For consistency, this should probably return an empty value.

Reply:
done